mvn_setup/tasks-parent/tasks-ui/src/core/TaskWork/TaskWorkFields.tsx

```diff
@@ -352,5 +368,66 @@ const CloseDialogButton: React.FC<{ onClose: () => void }> = ({ onClose }) => {
   )
 }
 
-const Fields = { Title, Description, Checklist, StartDate, DueDate, Attachments, Form, Messages, Menu, CloseDialogButton }
-export default Fields;
\ No newline at end of file
+const SplitButton: React.FC<{}> = () => {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef<HTMLDivElement>(null);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <React.Fragment>
+      <ButtonGroup variant="contained" ref={anchorRef}>
+        <Button startIcon={<CheckIcon />}>
+          <Typography><FormattedMessage id='core.taskWork.button.accept'></FormattedMessage></Typography>
+        </Button>
+        <Button size="small" onClick={handleToggle}>
+          <ArrowDropDownIcon />
+        </Button>
+      </ButtonGroup>
+      <Popper
+        open={open}
+        anchorEl={anchorRef.current}
+        transition
+        disablePortal
+        placement='top-end'
+      >
+        {({ TransitionProps }) => (
+          <Grow
+            {...TransitionProps}
+            style={{
+              transformOrigin: 'center bottom',
+            }}
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <MenuList autoFocusItem sx={{ textTransform: 'uppercase' }}>
+                  <MenuItem key='reject' onClick={() => handleClose}>
```

Comment:
It seems like this `<MenuItem><Icon><Typography><FormattedMessage>` pattern repeats 3 times in here.  Let's optimise it a bit.

Extract these to a separate React component so you can render them like this:

```
const SplitButton: React.FC<{}> = () => {
  const [open, setOpen] = React.useState(false);
  const anchorRef = React.useRef<HTMLDivElement>(null);
     ...
  return (
    <React.Fragment>
     ...
            <Paper>
              <ClickAwayListener onClickAway={handleClose}>
                <MenuList autoFocusItem sx={{ textTransform: 'uppercase' }}>
                  <MyButton buttonText='core.taskWork.button.reject' icon={<BlockIcon color='error' />} key='reject'  onClick={handleClose} />
                </MenuList>
              </ClickAwayListener>
            </Paper>
          </Grow>
        )}
      </Popper>
    </React.Fragment>
  );
}


```